src/tools/cephfs/cephfs-shell

```diff
@@ -865,14 +865,15 @@ class CephFSShell(Cmd):
         """
         return self.complete_filenames(text, line, begidx, endidx)
 
-    rmdir_parser = argparse.ArgumentParser(description='Remove Directory.')
-    rmdir_parser.add_argument('paths', help='Directory Path.', nargs='+',
+    rmdir_parser = argparse.ArgumentParser(description='Remove the '
+                                                       'directory(ies), '
+                                                       'if they are empty.')
```

Comment:
nit: better to move description to next line and fit it in a single line?

Reply:
Done